core/src/validator.rs

```diff
@@ -812,6 +812,12 @@ impl Validator {
             config.block_verification_method, config.block_production_method
         );
 
+        let (replay_vote_sender, replay_vote_receiver) = unbounded();
+
+        // block min prioritization fee cache should be readable by RPC, and writable by validator
+        // (by both replay stage and banking stage)
+        let prioritization_fee_cache = Arc::new(PrioritizationFeeCache::default());
+
```

Comment:
these two strictly-tx-execution related variables are hoisted here to be used `SchedulerPool` creation.